docs/application/dotnet/guides/machine-learning/pipeline.md

```diff
@@ -0,0 +1,180 @@
+# Pipeline
+
+You can use the `Tizen.MachineLearning.Inference.Pipeline` class to manage the topology of data and the interconnection between processors and models.
+
+Pipeline allows you to construct and execute a pipeline with multiple neural network models, multiple inputs and output nodes, multiple data processors, pre-and-post processors, and various data path manipulators.
+If the input is streamed data, Pipeline can simplify your application and improve its performance.
+
+The main features of Pipeline include:
+
+- [Pipeline construction](#construct)
+
+  You can compose the data stream pipeline using Machine Learning Inference with various elements of [GStreamer](https://gstreamer.freedesktop.org) and [NNStreamer](https://github.com/nnstreamer/nnstreamer).
+
+- [Push input data](#inputdata)
```

Comment:
Thank you for updating the headings. However, we do not need to change the bulleted list.
Articles and gerund format is fine here.
Therefore, can we revert back to the original sentence in the bulleted list?
Please verify.

For more information, see https://github.com/Samsung/tizen-docs/pull/1079#discussion_r412070796.

Reply:
Thanks, I just updated this PR. Please see the changes.

Reply:
Thank you for updating.